jaeger_client/tracer.py

```diff
@@ -51,8 +51,8 @@ class Tracer(opentracing.Tracer):
     def __init__(
         self,
         service_name: str,
-        reporter: BaseReporter,
-        sampler: Sampler,
+        reporter: 'BaseReporter',
+        sampler: 'Sampler',
```

Comment:
quotes not necessary with explicit imports, right?

Reply:
right